package.json

```diff
@@ -39,15 +39,18 @@
     "eslint-plugin-simple-import-sort": "6",
     "events": "3.3.0",
     "size-limit": "7.0.4",
+    "ts-node": "^10.4.0",
```

Comment:
should we try to keep dependencies versions fixed @jstashh ?

I know it is a requirement for yearn-finance-v3 repo